docs/pages/database-access/enroll-aws-databases/rds.mdx

```diff
@@ -388,21 +429,12 @@ $ tsh login --proxy=<Var name="example.teleport.sh:443" /> --user=alice
 $ tsh db ls
 # Name                           Description                                   Labels
 # ------------------------------ --------------------------------------------- --------
-# postgres-rds                   RDS instance in us-west-1                     ...
-# aurora-mysql                   Aurora cluster in us-west-1                   ...
-# aurora-mysql-custom-myendpoint Aurora cluster in us-west-1 (custom endpoint) ...
-# aurora-mysql-reader            Aurora cluster in us-west-1 (reader endpoint) ...
+# postgres-rds                                                                 ...
+# aurora-mysql                                                                 ...
+# aurora-mysql-custom-myendpoint                                               ...
+# aurora-mysql-reader                                                          ...
```

Comment:
nit: wonder if we should just keep the single database that's been configured and get rid other endpoints from discovery. also `tsh db ls` will "shrink" the description column when there is nothing. 

```suggestion
$ tsh db ls
# Name         Description Labels
# ------------ ----------- --------
# postgres-rds 
```